kinematic_plugins:
  search_libraries:
    - tesseract_kinematics_ur_factories
  inv_kin_plugins:
    manipulator:
      default: URInvKin
      plugins:
        URInvKin:
          class: URInvKinFactory
          config:
            base_link: base_link
            tip_link: tool0
            model: UR5
